src/base/string_list.hpp

```diff
@@ -197,7 +197,12 @@ class string_list_t {
       if (c == '"') {
         escaped_arg += "\\\"";
       } else if (c == '\\') {
+#ifdef _WIN32
+        // on Windows a backslash has semantic meaning and does not require escaping:
```

Comment:
For future reference (I hope to see more PR:s from you) please follow the code comment guidelines: https://github.com/mbitsnbites/buildcache/blob/master/doc/contributing.md#comments

In particular: Use proper sentences with capitalization.